scania-digital-design-system/sdds
# fix(dropdown): add modeVariant control

**Describe pull-request**  
Added modeVariant control to Native, Multiselect and Filter versions of the dropdown component.

**Solving issue**  
Fixes: [DTS-1127](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-1127)

**How to test**  
1. Go to Storybook link below
2. Check in Dropdown -> Native and Web Component Multiselect
3. Check in Dropdown Filter -> Default
4. Check that control "Mode variant" is present and works in both light and dark mode
5. Check for modeVariant property in Notes tab

[DTS-1127]: https://tegel.atlassian.net/browse/DTS-1127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
